src/AnomalyDetectors/OneClassSVM.php

```diff
@@ -182,7 +183,13 @@ public function train(Dataset $dataset) : void
             new SamplesAreCompatibleWithEstimator($dataset, $this),
         ])->check();
 
-        $this->model = $this->svm->train($dataset->samples());
+        $data = [];
+
+        foreach ($dataset->samples() as $i => $sample) {
+            $data[] = array_merge([1], $sample);
```

Comment:
I wonder which is faster ...

```php
array_merge([1], $sample)
```

or

```php
array_unshift(1, $sample)
```

From what I recall, unshift is linear because it needs to reindex the array or something. I think merge is also linear.

Reply:
Change is done ;)